src/ConnectedMode/UI/OrganizationSelection/OrganizationSelectionDialog.xaml.cs

```diff
@@ -20,15 +20,16 @@
 
 using System.Diagnostics.CodeAnalysis;
 using System.Windows;
+using SonarLint.VisualStudio.ConnectedMode.UI.Credentials;
 
 namespace SonarLint.VisualStudio.ConnectedMode.UI.OrganizationSelection;
 
 [ExcludeFromCodeCoverage]
 public partial class OrganizationSelectionDialog : Window
 {
-    public OrganizationSelectionDialog(IReadOnlyList<OrganizationDisplay> organizations)
+    public OrganizationSelectionDialog(IConnectedModeServices connectedModeServices, ICredentialsModel credentialsModel)
```

Comment:
I think it was simpler when it was just accepting a static list of connections, but I also get the point of moving the responsibility here

Reply:
Simpler, maybe. But that means we would have to load the connections from the dialog, which would violate the MVVM pattern: the view should have no knowledge about the Model (slCore in our case)